src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionWithFlavors.cs

```diff
@@ -50,7 +51,7 @@ public override bool HasActionSets
             // Light bulb will always invoke this property on the UI thread.
             AssertIsForeground();
 
-            if (_actionSets == null)
+            if (_actionSets.IsDefault)
```

Comment:
Why did the struct comparison to null warning not trigger for this?  Do we have it disabled in Roslyn?